src/components/shared/components/Sidebar/SideBar.scss

```diff
@@ -0,0 +1,6 @@
+@media (min-width: 900px) {
```

Comment:
Added so the content doesn't fall underneath the sticky navbar. More CSS will be needed